runtime/common/Cargo.toml

```diff
@@ -102,6 +102,7 @@ std = [
 	"xcm/std",
 ]
 runtime-benchmarks = [
+	"sp-api/disable-logging",
```

Comment:
Don't we need to enable this in other places as well?

Reply:
The common runtime is a dependency of all other runtimes. AFAIK this is the only place where we need to at it. It does not seem to be applicable to bridge and xcm runtimes because those don't depend on `sp-api`. Maybe someone else can step in with more knowledge about those runtimes.

Reply:
Sounds like we could have some kind of test to make sure we don't make this mistake again.

Reply:
You mean having logging enabled during benchmarking by accident?

Reply:
yes, but it is probably a PITA

Reply:
I have no idea on how to construct such a test other than some benchmark which will look for regressions. And something like that would be really nice to have anyways. After each benchmarks of the contracts pallet I need to manually look through a ton of benchmarks and scan for regressions. Would be nice to have this automated.